addons/mail/static/src/discuss/core/common/avatar_stack.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AvatarStack">
+        <div t-if="props.personas.length > 0">
+            <div t-if="!props.expandable" class="o-mail-AvatarStack py-1"><t t-call="mail.AvatarStack.collapsed"/></div>
+            <t t-else="">
+                <div class="d-flex justify-content-between text-muted cursor-pointer py-1 w-100" t-att-class="{'align-items-center': props.direction === 'horizontal'}" t-att-title="title" t-on-click="() => state.expanded = !state.expanded">
+                    <t t-if="state.expanded" t-call="mail.AvatarStack.expanded-item"><t t-set="persona" t-value="props.personas[0]"/></t>
+                    <t t-else="" t-call="mail.AvatarStack.collapsed"/>
+                    <i class="oi smaller ms-1" t-att-class="{'oi-chevron-down': !state.expanded, 'oi-chevron-up': state.expanded}"/>
```

Comment:
Chevron feels better closer to effective avatar stack content, and move on left:
<img width="288" alt="Screenshot 2024-09-27 at 16 03 03" src="https://github.com/user-attachments/assets/a2008e0e-faa4-43e4-9c9c-0dada4cee258">

Also when amount of members is low, the space on left feels wasted potential.
Showing the active speaker and/or label "in a call" would make UI feel more lively

Reply:
Moved on left. I'd rather improve this later for the "in call part". That's not that easy for the active speaker: it's a lot of movement in the UI. Imagine having several calls in the sidebar like we have on discord, it would really be awkward.

Reply:
The active speaker UI is worth when in the call, when not in call it's better to see something more static, e.g. "ongoing call"